src/plugins/engagement/public/components/iframe_chat/index.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+/* eslint-disable import/no-default-export */
+
+import { Chat } from './iframe_chat';
+export { Chat } from './iframe_chat';
+
+/**
+ * Exporting the Chat component as a default export so it can be loaded by React.lazy.
+ */
+export default Chat;
```

Comment:
NIT: not a problem, but not strictly required, see

https://github.com/elastic/kibana/blob/fb18ddb9786bb40cd712323caacfea3053d1ae71/x-pack/plugins/saved_objects_tagging/public/components/edition_modal/open_modal.tsx#L34-L36

Reply:
Hmm... I wonder which would be preferable from a holistic perspective.  I'll ping @elastic/kibana-operations for advice, perhaps.